src/components/react_multi_select.js

```diff
@@ -157,7 +157,8 @@ export default class ReactMultiSelect extends PureComponent {
       messages,
       listRowHeight,
       selectedListHeight,
-      dstItemsWrapperClassName
+      dstItemsWrapperClassName,
+        selectedItemFunc
```

Comment:
I have to admit, I'm not a fan of this prop name.
It's a component not a function.

How about **selectedItemRenderer** or selectedItemComponent